### ISSUE TITLE
Unwanted whitespaces and newline at the beginning of a snippet highlighting

### ISSUE THREAD

**Issue by vcarlucci** [Guyiming/Highlight.Blazor#2]
Hi, there seems to be a problem with the component "<CodeHighlight .. >". When I would highlight a snippet passed  by a property, the string will then shown with whitespaces at the beginning and it seems also it starts with a newline. Both, newline and whitespaces are not existing in the snippet string and wrong displayed.

Example code [page file]: **SnippetTest.razor**

```
@page "/SnippetTest"

<h3>SnippetTest</h3>

<CodeHighlight Indent=false CodeLanguage="csharp">@(Snippet.Trim())</CodeHighlight>

@code {
    public string Snippet { get; set; } = "public override void init()\n{\n   Backend.Init();\n}";
}
```

This code will output this:

<img width="468" alt="Screenshot 2023-09-05 091716" src="https://github.com/Guyiming/Highlight.Blazor/assets/33085651/3d5b05c4-52ff-418e-9cb7-6d02db538288">

*Image: you can see the red marked sections which represents the unwanted newline and whitespaces.

**Comment by Guyiming** [comment]
fixed in 0.1.10

**Comment by vcarlucci** [comment]
@Guyiming, tested and is working now with 01.1.10

Thanks!
